contracts/adapters/stake-dao/StakeDAOTokenAdapter.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: None
+
+pragma solidity 0.6.5;
+pragma experimental ABIEncoderV2;
+
+import {ERC20} from "../../ERC20.sol";
+import {TokenMetadata, Component} from "../../Structs.sol";
+import {TokenAdapter} from "../TokenAdapter.sol";
+
+interface Vault {
+    function token() external view returns (address);
+
+    function getPricePerFullShare() external view returns (uint256);
+}
+
+/**
+ * @title Token adapter for Stake DAO Vaults.
+ * @dev Implementation of TokenAdapter abstract contract.
+ * @author Elephant memory/strength
+ */
+contract StakeDaoTokenAdapter is TokenAdapter {
+    address public sdveCrv = 0x478bBC744811eE8310B461514BDc29D03739084D;
```

Comment:
we usually save such addresses as `address internal constant`

Reply:
it might have a new address in the future, hence not a constant